lib/ansible/plugins/connection/docker.py

```diff
@@ -132,6 +135,23 @@ def exec_command(self, cmd, in_data=None, sudoable=False):
         stdout, stderr = p.communicate(in_data)
         return (p.returncode, stdout, stderr)
 
+    def _build_exec_cmd(self, cmd):
```

Comment:
Ansible convention has methods and functions that are called internally are near the top of the class.